### PR TITLE
[Aptos Framework] Add events for Stake module operations

### PR DESCRIPTION
**Pull request by movekevin**
### Description

This PR adds events for all different Stake operations

### CODE REVIEW

File: aptos-move/framework/aptos-framework/sources/configs/Stake.move [hunk 1]
```diff
@@ -161,6 +161,70 @@ module AptosFramework::Stake {
...
+
+    struct SetOperatorEvent has drop, store {
+        pool_address: address,
+        old_operator: address,
+        new_operator: address,
+    }
+
+    struct AddStakeEvent has drop, store {
+        pool_address: address,
+        amount: u64,
```
Comment (zekun000) [inline aptos-move/framework/aptos-framework/sources/configs/Stake.move:190]: the name should be clearer about whether it's the total amount or the added amount

**Review by zekun000 — APPROVED** [review]
I guess we don't care about the parallelism of those transactions, otherwise we'll need to put events in individual address

**Comment by movekevin** [comment]
> I guess we don't care about the parallelism of those transactions, otherwise we'll need to put events in individual address

Yes these events are per stake pool. There can't be that many operations for a single stake pool

File: aptos-move/framework/aptos-framework/sources/configs/Stake.move (hunk 1 above)
Comment (movekevin) [reply aptos-move/framework/aptos-framework/sources/configs/Stake.move]: Updated

**Pull request closed without merging**